views/add_form.erb

```diff
@@ -0,0 +1,15 @@
+
+<link rel="stylesheet" href="style.css">
+
+<body>
+
+<h1>Add Peep</h1>
+
+<form action=/add_submit method='post'>
+    Enter Your Peep: (240 char max)
+  <textarea name="message" rows="10" cols="30" maxlength=240>
+  </textarea>
+  <input type='submit' name="Post Peep">
```

Comment:
There is a space in the text area when you go to write a message, I can't quite work out where it's coming from but I think it must be on this bit somewhere